google/resource_logging_billing_account_sink.go

```diff
@@ -0,0 +1,75 @@
+package google
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceLoggingBillingAccountSink() *schema.Resource {
+	schm := &schema.Resource{
+		Create: resourceLoggingBillingAccountSinkCreate,
+		Read:   resourceLoggingBillingAccountSinkRead,
+		Delete: resourceLoggingBillingAccountSinkDelete,
+		Update: resourceLoggingBillingAccountSinkUpdate,
+		Schema: resourceLoggingSinkSchema(),
```

Comment:
Import support? You have all the context you need fresh in your mind. Since this PR is long, I think it is a good idea to do it in a separate PR.

Reply:
Agreed! I actually started adding that but stopped as it got pretty complicated. My approach was modeled on yours